k8s/demo/common/em-ccd-orchestrator/em-ccd-orchestrator-demo.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-ccd-orchestrator-demo
+  namespace: em-ccd-orchestrator
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: em-ccd-orchestrator-demo
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-ccd-orchestrator
+    version: 0.1.4
+  values:
+    ### EM-CCD-ORCHESTRATOR SERVICES ###
+    em-stitching:
+      java:
```

Comment:
This is nested inside an alias

Reply:
I have seen this done in most other charts in Demo -> see `k8s/demo/common/divorce/div-demo.yaml`and `k8s/demo/common/ccd/latest-ccd-chart.yaml`